src/aws/osml/model_runner/database/feature_table.py

```diff
@@ -89,14 +89,21 @@ def add_features(self, features: List[Feature], metrics: MetricsLogger = None):
         """
         if isinstance(metrics, MetricsLogger):
             metrics.set_dimensions()
+            metrics.put_dimensions(
+                {
+                    MetricLabels.OPERATION_DIMENSION: MetricLabels.FEATURE_STORAGE_OPERATION,
+                }
+            )
+            metrics.put_metric(MetricLabels.INVOCATIONS, 1, str(Unit.COUNT.value))
+
         start_time_millisec = int(time.time() * 1000)
         # These records are temporary and will expire 24 hours after creation. Jobs should take
         # minutes to run, so this time should be conservative enough to let a team debug an urgent
         # issue without leaving a ton of state leftover in the system.
         expire_time_epoch_sec = Decimal(int(start_time_millisec / 1000) + (2 * 60 * 60))
         with Timer(
             task_str="Add image features",
-            metric_name=MetricLabels.FEATURE_STORE_LATENCY,
+            metric_name=MetricLabels.DURATION,
```

Comment:
Yeah this was a nice quality of life improvement generalizing the duration and error names.